grpc/test_support/src/internal/mod.rs

```diff
@@ -37,7 +37,9 @@ pub use wasm_test_builder::{
 
 pub const DEFAULT_VALIDATOR_SLOTS: u32 = 5;
 pub const DEFAULT_AUCTION_DELAY: u64 = 3;
-pub const DEFAULT_LOCKED_FUNDS_PERIOD: EraId = 15;
+/// Default lock-in period of 90 days
+/// <https://www.wolframalpha.com/input/?i=convert+90+days+to+milliseconds>
+pub const DEFAULT_LOCKED_FUNDS_PERIOD_MILLIS: u64 = 7776000000;
```

Comment:
You could spell out the 90 days as a product:
```suggestion
pub const DEFAULT_LOCKED_FUNDS_PERIOD_MILLIS: u64 = 90 * 24 * 60 * 60 * 1000;
```

Reply:
Good suggestion, will do.

Reply:
Donezo.